modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase

```diff
@@ -616,4 +616,124 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-9794 | As a Admin I want to be able to view the Setting Tab on Order Item from a virtual product"
+	@priority = 4
+	test CanViewVirtualProductSettingsOnOrderItemPage {
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And a business account with a buyer user are created") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And a virtual product is created") {
+			CommerceJSONProductsAPI._addCommerceProduct(
+				catalogName = "Minium",
+				gtin = "GTIN1",
+				manufacturerPartNumber = "MPN1",
+				productDescription = "Full Description",
+				productName = "Virtual Product",
+				productType = "Virtual",
+				shortDescription = "Short Description",
+				sku = "VIRTUALSKU");
```

Comment:
@Brunno0liveira why do you need `gtin` and `manufacturerPartNumber` fields here?